hazelcast/src/main/java/com/hazelcast/map/impl/operation/DefaultMapOperationProvider.java

```diff
@@ -171,8 +171,8 @@ public MapOperation createTxnDeleteOperation(String name, Data dataKey, long ver
 
     @Override
     public MapOperation createTxnLockAndGetOperation(String name, Data dataKey, long timeout, long ttl, String
-            ownerUuid) {
-        return new TxnLockAndGetOperation(name, dataKey, timeout, ttl, ownerUuid);
+            ownerUuid, boolean loadKey) {
```

Comment:
Forget about the read-through. It would suggest that even if the data would be in memory, the data is still loaded from the db. But we do need to find a more self explaining name than 'loadKey'.